tools/gnali/gnali.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="gnali" name="gNALI" version="1.0.3" python_template_version="3.7">
+<tool id="gnali" name="gNALI" version="1.0.4" python_template_version="3.7">
```

Comment:
you could use here the @VERSION@ macro as well :)

Reply:
Thanks for the suggestion! Implemented in f1723f9